### ISSUE TITLE
(macOS 10.13) Cannot get OpenGL function pointer!

### ISSUE THREAD

**Issue by simon300000** [lhc70000/iina#971]
**System and IINA version:**

- macOS 10.13 Beta (17A330h)
- IINA 0.0.12 build 37

**Expected behavior:**
Showing: opening a mkv showing "Cannot get OpenGL function pointer!" on a prompt and close IINA itself. No system crash report.

- Crash report:
```
I could give Spindump if needed?
```

- MPV log:
```
[   0.011][v][cplayer] mpv 0.25.0 (C) 2000-2017 mpv/MPlayer/mplayer2 projects
[   0.011][v][cplayer]  built on Tue Apr 25 19:22:08 HKT 2017
[   0.011][v][cplayer] ffmpeg library versions:
[   0.011][v][cplayer]    libavutil       55.58.100
[   0.011][v][cplayer]    libavcodec      57.89.100
[   0.011][v][cplayer]    libavformat     57.71.100
[   0.011][v][cplayer]    libswscale      4.6.100
[   0.011][v][cplayer]    libavfilter     6.82.100
[   0.011][v][cplayer]    libswresample   2.7.100
[   0.011][v][cplayer] ffmpeg version: 3.3
[   0.011][v][cplayer] 
[   0.011][v][cplayer] Configuration: waf configure --prefix=/usr/local/Cellar/mpv/0.25.0 --enable-zsh-comp --enable-libmpv-shared --enable-html-build --confdir=/usr/local/etc/mpv --datadir=/usr/local/Cellar/mpv/0.25.0/share/mpv --mandir=/usr/local/Cellar/mpv/0.25.0/share/man --docdir=/usr/local/Cellar/mpv/0.25.0/share/doc/mpv --zshdir=/usr/local/Cellar/mpv/0.25.0/share/zsh/site-functions --enable-libbluray
[   0.011][v][cplayer] List of enabled features: 52 apple-remote asm atomics avutil-imgcpy-uc bsd-fstatfs build-date c11-tls cocoa coreaudio cplayer cuda-hwaccel debug-build dlopen encoding fchmod gcc-tls gl gl-cocoa glob gnuc iconv is_ffmpeg jpeg lcms2 libass libass-osd libav libavcodec libavdevice libbluray libdl libm libmpv-shared lua nanosleep optimize osx-thread-name plain-gl posix posix-or-mingw posix-spawn pthreads shm stdatomic subprocess termios tv uchardet videotoolbox-gl videotoolbox-hwaccel zlib zsh-comp
[   0.013][v][bdmv/bluray] Opening /Applications/IINA.app/Contents/Resources/config/iina-default-input.conf
[   0.013][v][file] Opening /Applications/IINA.app/Contents/Resources/config/iina-default-input.conf
[   0.013][v][file] Stream opened successfully.
[   0.013][v][input] Parsing input config file /Applications/IINA.app/Contents/Resources/config/iina-default-input.conf
[   0.013][v][input] Input config file /Applications/IINA.app/Contents/Resources/config/iina-default-input.conf parsed: 71 binds
[   0.013][v][global] user path: '/Users/simon3000/Library/Application Support/com.colliderli.iina/log/2017-8-16-0-13-21_H7z5dZ.log' -> '/Users/simon3000/Library/Application Support/com.colliderli.iina/log/2017-8-16-0-13-21_H7z5dZ.log'
[   0.013][v][ytdl_hook] Loading lua script @ytdl_hook.lua...
[   0.013][v][ytdl_hook] loading mp.defaults
[   0.014][v][ytdl_hook] loading @ytdl_hook.lua
[   0.014][v][cplayer] Run command: hook-add, flags=0, args=[on_load, 1, 10]
[   0.014][v][cplayer] Run command: hook-add, flags=0, args=[on_preloaded, 2, 10]
[   0.015][v][cplayer] Done loading @ytdl_hook.lua.
[   0.015][v][osx] Exiting...
[   0.015][v][cplayer] Set property: vo="opengl-cb" -> 1
[   0.015][v][cplayer] Set property: keepaspect="no" -> 1
[   0.015][v][cplayer] Set property: opengl-hwdec-interop="auto" -> 1
```

**Steps to reproduce:**
Open IINA
Open the mkv video I played

- [x] MPV does not have this problem.
- MPV version that works
```
mpv 0.25.0 (C) 2000-2017 mpv/MPlayer/mplayer2 projects
 built on Wed Aug 16 00:31:08 CST 2017
ffmpeg library versions:
   libavutil       55.58.100
   libavcodec      57.89.100
   libavformat     57.71.100
   libswscale      4.6.100
   libavfilter     6.82.100
   libswresample   2.7.100
ffmpeg version: 3.3.2
```

**How often does this happen?**
All the time, I tried with this video, other .mkv, a .mov, a .m4v, which all work with VLC.

**Comment by saagarjha** [comment]
Duplicate of #948. Can you try reinstalling macOS and seeing if the error persists?

**Comment by simon300000** [comment]
@saagarjha Yes I think this would work, I'm trying it now since lot of other application not working like Safari and App Store after this update, thank you

**Comment by simon300000** [comment]
It seems to be a macOS problem, problem solved after update to 17A344b using softwareupdate
